### DISCUSSION TITLE
Java 8 support removed and CI/CD process simplified and future proofed

### DISCUSSION THREAD

**Discussion by jarlah** [jarlah/testcontainers-ceph#54] [Announcements]
New version 2.0.0 released in maven central and this process has now been greatly simplified. 

Its a very simple package, but with dependabot updates and continuous usage it will remain maintained and up to date.

@brice-laurencin could you see if it still works with the new version 2.0.0 ? It might take some time before its available in maven central.

**Comment by brice-laurencin** [comment]
Hey @jarlah!
Thanks a lot for this.
It seems to be working perfectly!
